metals/src/main/scala/scala/meta/internal/metals/doctor/ProblemResolver.scala

```diff
@@ -188,11 +178,12 @@ class ProblemResolver(
       mtagsResolver.isSupportedScalaVersion(version)
 
     val scalaVersionProblem = scalaTarget.scalaVersion match {
-      case version if !isSupportedScalaVersion(version) && scalaTarget.isSbt =>
```

Comment:
So this was actually done intentionally to check for Scala version. Metals knows which Scala versions it supports, but it doesn't know anything about sbt versions.

However, if sbt version uses a Scala version that Metals supports then  we also support that sbt version. The same with everything else.